services/socket.js

```diff
@@ -8,8 +9,27 @@ app.io = require("socket.io")({
 
 app.io.on("connection", (socket) => {
   socket.on("user-send", (data) => {
-    console.log("socket.on ~ data", data);
-    app.io.emit("broadcast", data);
+    if (data[data.length - 1] === "s") {
+      const { x: xPosition, y: yPosition } = robot.getMousePos();
+
+      robot.moveMouse(xPosition, yPosition + 5);
+      app.io.emit("broadcast", data);
+    } else if (data[data.length - 1] === "a") {
+      const { x: xPosition, y: yPosition } = robot.getMousePos();
+
+      robot.moveMouse(xPosition - 5, yPosition);
+      app.io.emit("broadcast", data);
+    } else if (data[data.length - 1] === "d") {
+      const { x: xPosition, y: yPosition } = robot.getMousePos();
+
+      robot.moveMouse(xPosition + 5, yPosition);
+      app.io.emit("broadcast", data);
+    } else if (data[data.length - 1] === "w") {
+      const { x: xPosition, y: yPosition } = robot.getMousePos();
+
+      robot.moveMouse(xPosition, yPosition - 5);
+      app.io.emit("broadcast", data);
+    }
```

Comment:
```suggestion
app.io.on("connection", (socket) => {
  socket.on("user-send", (data) => {
    const { x: xPosition, y: yPosition } = robot.getMousePos();

    if (data === "down") {
      robot.moveMouse(xPosition, yPosition + 10);
    } else if (data === "left") {
      robot.moveMouse(xPosition - 10, yPosition);
    } else if (data === "right") {
      robot.moveMouse(xPosition + 10, yPosition);
    } else if (data === "up") {
      robot.moveMouse(xPosition, yPosition - 10);
    } else if (data === "click") {
      robot.mouseClick();
    }

    app.io.emit("broadcast", data);
  });
});
```
위의 코드처럼 `app.io.emit("broadcast", data);`라는 중복된 분을 따로 빼주면 좋습니다.